src/main/java/salesforce/classic/WorkTypeClassicPage.java

```diff
@@ -0,0 +1,35 @@
+package salesforce.classic;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import salesforce.BasePage;
+
+
+public class WorkTypeClassicPage extends BasePage {
+    @FindBy(name = "new")
+    protected WebElement newBtn;
+
+    @FindBy(css = "#tryLexDialogX")
+    protected WebElement closeAnnouncement;
+
+    public WorkTypeClassicPage(final WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
+    protected void waitForPageLoaded() {
+        System.out.println("Entrooo");
+        wait.until(ExpectedConditions.visibilityOf(newBtn));
+    }
+
+    public void closeAnnouncement() {
```

Comment:
Perhaps, this method could be used for other features, maybe it would be better to define it in more generic class?